storybookjs/storybook
# Attempt to make propTablesExclude usage clearer

See issue #2266, specifically https://github.com/storybooks/storybook/issues/2266#issuecomment-351472412 for discussion.

Issue: #2266 

## What I did
Added text to documentation make propTablesExclude clearer

## How to test
No changes to code made.

Is this testable with jest or storyshots?
No

Does this need a new example in the kitchen sink apps?
No but would maybe be a good idea.

Does this need an update to the documentation?
This is an update.

If your answer is yes to any of these, please make sure to include it in your PR.
